test-plugins/org.yakindu.sct.model.stext.test/src/org/yakindu/sct/model/stext/test/validation/STextJavaValidatorTest.java

```diff
@@ -652,6 +652,19 @@ public void checkUnusedEntry() {
 		assertIssueCount(diagnostics, 1);
 		assertWarning(diagnostics, ENTRY_UNUSED);
 	}
+	
+	@Test
+	public void checkNotRaisedOutEvent() {
+		statechart = AbstractTestModelsUtil.loadStatechart(VALIDATION_TESTMODEL_DIR + "NeverRaisedOutEvent.sct");
+		Diagnostic diagnostics = Diagnostician.INSTANCE.validate(statechart);
+	
+		// used as transition trigger and inside reaction guard
+		assertWarning(diagnostics, String.format(OUT_EVENT_NEVER_RAISED, "B"));
+		
+		// used inside reaction guard
+		assertWarning(diagnostics, String.format(OUT_EVENT_NEVER_RAISED, "A"));
+		assertIssueCount(diagnostics, 4);
```

Comment:
You have 4 issues but only check two of them, is this fine or is there something missing?